0000-product.md

```diff
@@ -0,0 +1,463 @@
+- Feature Name: Product
+- Start Date: 5/13/19
+- RFC PR: [#5](https://github.com/hyperledger/grid-rfcs/pull/5)
+
+# Summary
+[summary]: #summary
+
+
+This RFC proposes a generic and extensible framework for a Hyperledger Grid
+Product entity as well as a more specific - and fully encapsulated - GS1
+compliant Product.
+
+GS1 is a widely used data standard in enterprises and, given that positioning
+and familiarity, Grid support feels natural.  Additional implementations of
+Product for specialized industries or use cases may derive from or extend this
+implementation.
+
+_For the base implementation of a product on grid, there will be 4 fields. A
+product_id, a product_namespace, an owner, and properties (a repeated key-value
+field)._
+
+
+# Motivation
+[motivation]: #motivation
+
+The Grid Product implementation is designed for sharing product master data
+between participants. Product is a near universal concept within supply chain
+solutions and would naturally be one of the highest areas of re-use across Grid
+applications.
+
+The desgn will address use cases including:
+
+- Sharing of Product master data across a network
```

Comment:
Is this part of Product or Catalog?

Reply:
This is a part of Product, I have a draft PR up for Catalog here: https://github.com/hyperledger/grid-rfcs/pull/14